packages/fleather/example/pubspec.yaml

```diff
@@ -15,6 +15,8 @@ dependencies:
   image_picker: ^1.0.7
   fleather:
     path: ../
+  parchment:
```

Comment:
Why do we need this?

Reply:
Analyzer started warning about usage of `import 'package:parchment/codecs.dart';` in `editing_test.dart`. @amantoux